common/tools/dev-tool/src/commands/run/update-snippets.ts

```diff
@@ -285,12 +287,12 @@ async function parseSnippetDefinitions(
       )
         .replace(
           // Need to get rid of any ts-ignores that were added because of unused symbols
-          /\r?\n[ ]*\/\/\s*@ts-ignore\s*\r?\n/g,
+          TS_IGNORE,
```

Comment:
We probably want to use `\n` instead of `EOL` to ensure consistency in source code across platforms

Reply:
Done